CMakeLists.txt

```diff
@@ -787,7 +787,8 @@ target_compile_definitions(_C PRIVATE CUTLASS_ENABLE_DIRECT_CUDA_DRIVER_CALL=1)
 set(VLLM_MOE_EXT_SRC
   "csrc/moe/torch_bindings.cpp"
   "csrc/moe/moe_align_sum_kernels.cu"
-  "csrc/moe/topk_softmax_kernels.cu")
+  "csrc/moe/topk_softmax_kernels.cu"
+  "csrc/moe/grouped_topk_kernels.cu")
```

Comment:
Move this kernel to the CUDA section below

Reply:
Good catch! I have moved this. Thanks.